python/core/qgsdistancearea.sip

```diff
@@ -146,6 +146,21 @@ class QgsDistanceArea
      */
     double measureLine( const QgsPoint& p1, const QgsPoint& p2 ) const;
 
+    /**
+     * calculates distance from one point with distance in meters and azimuth (direction)
+     * based on inverse Vincenty's formula when Geographic
+     * otherwise QgsPoint.project() will be called after QgsUnitTypes::fromUnitToUnitFactor() has been applied to the distance
+     * note: usage:
+     *  QgsDistanceArea myDa;
+      * myDa.setSourceCrs( layer->crs() );
+     * \since QGIS 3.0
+     * @param p1 start point [can be Cartesian or Geographic]
+     * @param distance expected to be in meters
+     * @param projected_point calculated projected point
+     * @return distance in mapUnits
+     */
+    double measureLineProjected( const QgsPoint &p1, double distance = 1, double azimuth = M_PI / 2, QgsPoint *projectedPoint /In,Out/ = nullptr ) const;
```

Comment:
The c++ code will only fill the result if it being passed in the function with a non-null value
* so how will it know to do this?
   * does python add this when the tupel contains a second value?